### ISSUE TITLE
Add dimmers, thermostat, ... to nhc-homekit

### ISSUE THREAD

**Issue by janstp** [kevinbungeneers/nhc-homekit#4]
Hi Kevin,

First of all, nice work here. I got this working at my home and can now control my lights.
But off-coarse it would be nice if I could also control my dimmers, motors, ...

I'am pretty new to nodejs and github, so I'm not an expert at all, but as far I can see, the nhc-homekit can now only support on/off ( true/false ) values.
I just wondered if you already did extra work here, before I try to make the changes myself to allow 0-100 values too? In that way it would be possible to control the dimmers.

And off-coarse I would then also need to add the actions of type 2 ( dimmers ), type 4 ( motors ), ...

Kind regards,

Jan

**Comment by extric99** [comment]
I agree that getting the dimmers and motors working would be awesome. @kevinbungeneers If code need to be tested or if need a spare motormodule to write it I am happy to lend you one.

**Comment by kevinbungeneers** [comment]
Hi guys!

I'm afraid that there's not much that I can do at this moment, seeing I don't have access to an NHC installation anymore 😞 

If someone decides to try to develop those features themselves, I'd be happy to assist wherever I can though!
